instrumentation-core/servlet-2.2/src/main/java/io/opentelemetry/instrumentation/servlet/ServletHttpServerTracer.java

```diff
@@ -107,4 +125,13 @@ protected String flavor(HttpServletRequest connection, HttpServletRequest reques
   protected String requestHeader(HttpServletRequest httpServletRequest, String name) {
     return httpServletRequest.getHeader(name);
   }
+
+  private static String getSpanName(HttpServletRequest request) {
+    String spanName = request.getServletPath();
+    String contextPath = request.getContextPath();
+    if (contextPath != null && !contextPath.isEmpty() && !contextPath.equals("/")) {
+      spanName = contextPath + spanName;
```

Comment:
Technically, this is a duplication with `ServletContextPath.prepend`

Reply:
ya, I could add `public static String prepend(String contextPath, String spanName)` to `ServletContextPath`, but I'm thinking better to live with the duplication instead of adding API surface to an already confusing class `ServletContextPath`